_example/example.go

```diff
@@ -14,7 +14,9 @@ func main() {
 		Name: "Lamp",
 	}
 	sw := accessory.NewSwitch(switchInfo)
-	t, err := hap.NewIPTransport("00102003", sw.Accessory)
+
+	config := hap.Config{Pin: "12344321", Port: ":12345", StoragePath: "db"}
```

Comment:
Should the port maybe not include the colon before the numbers?   ":12345" should maybe be "12345", or even an integer, 12345?